examples/klotho-analysis/klotho-go-example/README.md

```diff
@@ -0,0 +1,69 @@
+# Simple Klotho app
+
+## Install Klotho
+
+### In linux
+
+Run
+
+```bash
+curl -fsSL "http://srv.klo.dev/update/latest/linux/x86_64?stream=pro:latest" -o klotho
+```
+
+And then:
+
+```bash
+sudo chmod +x klotho
```

Comment:
as long as the Klotho binary is located under any user's folder, sudo cmd is not required here